labelbox/client.py

```diff
@@ -1786,3 +1789,17 @@ def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
             experimental=True)["project"]["batches"]["nodes"][0]
 
         return Entity.Batch(self, project_id, batch)
+
+    def run_foundry_app(self, model_run_name: str, data_rows: Union[DataRowIds,
```

Comment:
```suggestion
    def run_foundry_app(self, model_run_name: str, data_rows: DataRowIdentifiers,
```

Reply:
I didn't use `DataRowIdentifiers` to make it clearer in docs and IDE code reference how this function can be used. i.e. if I see DataRowIdentifers in docs, I have to first find what it is, I will see" `Union[UniqueIds, GlobalKeys]`, its not clear here that `UniqueIds` means DataRowIds. Also, now I need to know about symbols two level-deep, first `DataRowIdentifers` and then one of `UniqueIds` or `GlobalKeys`. I agree with your suggestion for internal code, but IMO, for a public-facing method, the current annotation is clearer.